src/tactic/doc_commands.lean

```diff
@@ -428,6 +424,16 @@ add_tactic_doc
   decl_names := [`tactic.interactive.simp],
   tags := ["core", "simplification"] }
 
+/--
+Accepts terms with the type `component tactic_state string` or `html empty` and
```

Comment:
@EdAyers can you add to this tactic doc or give some suggestions?

Reply:
Maybe just add that it needs a compatible version of vscode (that isn't released yet)